data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -99,7 +99,7 @@
       <xen_append>splash=silent quiet console=<%= $get_var->('SERIALDEV') %>,115200</xen_append>
       <xen_kernel_append><%= $check_var->('IPMI_HW', 'AMD') ? "" : "dom0_max_vcpus=4 dom0_mem=8192M,max:8192M" %> vga=gfx-1024x768x16 loglvl=all guest_loglvl=all</xen_kernel_append>
       % } else {
-      <append>splash=silent</append>
+      <append>splash=silent <%= $check_var->('IPMI_HW', 'AMD') ? "" : "intel_iommu=on" %></append>
```

Comment:
This line is put in host_15 only. How is it ensured on host_12?

Reply:
No such requirement for sles12 to test sriov.

Reply:
Oh, okay, did not notice this difference before. @Julie-CAO  Are we handling this way for developing test matrix?

Reply:
Yes, we removed `intel_iommu=on` on sle12sp5 host autoyast template in my previous PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/b32cb47c52ab0ac13e1b64c5f137b1fc4d284c60

Reply:
Good, thank you all.